packages/bierzo-wallet/src/routes/balance/components/index.tsx

```diff
@@ -15,6 +16,7 @@ interface Props {
   readonly onSendPayment: () => void;
   readonly onReceivePayment: () => void;
   readonly onRegisterUsername: () => void;
+  readonly rpcEndpointType: RpcEndpointType | undefined;
```

Comment:
Why do we allow undefined? If we want to allow undefined, let's make it optional such that you can create the component without setting the value